jsonrpc-parent/jsonrpc-documentation/src/main/java/io/yupiik/uship/jsonrpc/doc/openapi/OpenRPC2OpenAPI.java

```diff
@@ -154,7 +154,7 @@ protected String toErrorName(final OpenRPC.ErrorValue error, final int counter)
         return error.getCode() >= 100 && error.getCode() <= 999 ?
                 String.valueOf(error.getCode()) :
                 // not very great but fits swagger UI, 5xx means error in this convention, code is in description
-                String.format("5%02d", error.getCode() < 100 ? error.getCode() : counter);
+                String.format("5%02d", 0 < error.getCode() && error.getCode() < 100 ? error.getCode() : counter);
```

Comment:
think we now miss the case the code is < 0 in swagger-ui, how does it render? from my last tests it does not validate the openapi

Reply:
yes does not validate when code isn't a valide html status code. So fix like this

1. 100 <= code <=999 -> valide code
2. 0 < code < 100 -> 5XX where X is code value (01 for code = 1)
3. else 5XX where X is counter (won't work if counter >= 100)

Reply:
got it, please just ensure the actual code is still somewhere in the doc - even if the description - to ensure an user can rely on the swagger-ui rendering

:+1: